sigar_linux.go

```diff
@@ -155,6 +155,36 @@ func (self *FileSystemList) Get() error {
 	return err
 }
 
+func (self *DiskList) Get() error {
```

Comment:
Is the `DiskList` being an alias for a list a common pattern already in this lib?